README.md

```
@@ -1,47 +1,94 @@
->   - Docker – [_How to do it?_](#-docker)
->   - Firebase function
->   - AWS λ (lambda function)
->   - CloudFlare Worker
->   - Chrome extension
->   - React native app
->   - Browsers (Pay attention to CORS)
-
-## 🗜️ Install
+# 🎬 CSFD API
+
+### Modern TypeScript library for Czech Movie Database (ČSFD.cz)
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)